static/js/apps/nl_interface/nl_constants.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Constants and functions shared across components.
+ */
+
+export const NL_URL_PARAMS = {
+  IDX: "idx",
+  LLM: "llm",
+  SMALL: "small",
```

Comment:
I would make SMALL and MEDIUM separate constants and just keep the url param keys in here. That way it'll be easy to tell what url param keys are available

Reply:
ok, changed